client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -79,9 +84,13 @@ function postChallenge(update, username) {
         updateComplete(),
         submitChallengeComplete()
       ];
-      // TODO(Post-MVP): separate endpoint for trophy submission?
-      if (isTrophyMissing)
-        actions.push(createFlashMessage(trophyMissingMessage));
+
+      if (challengeType === challengeTypes.msTrophy && type === 'error') {
+        actions = [createFlashMessage(msTrophyError)];
```

Comment:
Submitting challenge will be disabled if the trophy fails to submit.

```suggestion
        actions = [createFlashMessage(msTrophyError), submitChallengeError()];
```

Reply:
nice catch @Sboonny 👍